doc/polyglossia.tex

```diff
@@ -771,7 +771,170 @@ \subsection{italian}\label{italian}
+	IVLIVS & IVLIVS & IULIUS & IULIUS \\
+	\hline
+	\end{tabular}
+	\caption{\label{tab:latin-spelling}Spelling differences between the Latin
+	language variants. The capitalization of month names and the use of
+	\emph{i/j} may be affected by the \texttt{capitalizemonth} and the
+	\texttt{usej} option.}
+	\end{table}
+	\item \TB{hyphenation} = classic, modern, or liturgical
+
+	There are three different sets of hyphenation patterns for Latin. Separate
+	documention for them is available on the
+	internet.\footnote{\url{https://github.com/gregorio-project/hyphen-la/blob/master/doc/README.md\#hyphenation-styles}}
+	Each of the four variants mentioned above has its default set of hyphenation
+	patterns as indicated by table \ref{tab:latin-hyphenation}. Use the
+	\texttt{hyphenation} option if the default style does not fit your needs.
+	Note that the liturgical hyphenation patterns are the default of none of the
+	language variants. To use them, you have to say
+	\texttt{hyphenation=liturgical} in any case.
+	\begin{table}
+	\centering
+	\begin{tabular}{ll}
+	\hline
+	\emph{Language variant} & \emph{Default hyphenation style} \\
+	\hline
+	classic & classic \\
+	medieval & modern \\
+	modern & modern \\
+	ecclesiastic & modern \\
+	\hline
+	\end{tabular}
+	\caption{\label{tab:latin-hyphenation}Latin default hyphenation styles}
+	\end{table}
+	\item \TB{ecclesiasticfootnotes} = true or \emph{false}
+
+	Use footnotes as provided by the \pkg{ecclesiastic} package, which typesets
+	footnotes with ordinary instead of superior numbers and without indentation.
+	As many ecclesiastic documents and liturgical books use footnotes that are
+	very similar to the ordinary \LaTeX{} ones, we do not use this footnote style
+	as default even for the \texttt{ecclesiastic} variant.
+
+	Note that this option is only possible if Latin is the main language of your
+	document.
+	\item \TB{usej} = true or \emph{false}
+
+	Use \emph{J/j} in predefined terms. The letter \emph{j} is not of ancient
+	origin. In early modern times, it was used to distinguish the consonantic
+	\emph{i} from the vocalic~\emph{i}. Nowadays, the use of \emph{j} has
+	disappeared from most Latin publications. So \emph{false} is the default
+	value for all four language variants. Use this option if you prefer
+	\emph{Januarii} and \emph{Maji} to \emph{Ianuarii} and \emph{Maii}.
+	\item \TB{capitalizemonth} = true or false
+
+	Capitalize the month name when printing dates (using the \cmd\today{}
+	command).  Traditionally, month names are capitalized. However, in recent
+	liturgical books they are lowercase. So \emph{true} is the default value for
+	the variants \texttt{classic}, \texttt{medieval}, and \texttt{modern},
+	whereas \emph{false} is the default value for the \texttt{ecclesiastic}
+	variant.
+	\item \TB{babelshorthands} = true or \emph{false}
+
+	Enable shorthands inherited from \pkg{babel-latin} and the \pkg{ecclesiastic}
+	package as explained in table
+	\ref{tab:latin-shorthands}.
```

Comment:
Note that I have introduced a `shorthands` environment for shorthand documentation. For consistency reasons, please use that rather than a table

Reply:
Yes, of course. It's done.